charts/hivemq-platform-operator/values.yaml

```diff
@@ -42,6 +42,18 @@ serviceAccount:
   # The service account name to be used, or "hivemq-<release name>" if not set.
   name: ""
 
+# Configures the service account and permissions the operator creates for the HiveMQ Platform Pods.
+platformServiceAccount:
+  # Specifies whether a service account should be created for each HiveMQ Platform.
+  create: true
+  # Specifies whether the service account for the HiveMQ Platform should be validated.
```

Comment:
Would it make sense to allow a custom service account to be set here just like the above "serviceAccount" setting. Than both types of serviceAccounts are providing the same options.

Reply:
We would need to add another configuration option to the operator then, which would change the default name of the service account for all reconciled platforms. If the platform would configure the service account name, this would then override this setting.

Priority (highest first):
* HiveMQ Platform Helm chart: `nodes.serviceAccount`
* HiveMQ Platform Operator Helm chart: `hivemqPlatformServiceAccount.name`
* HiveMQ Platform Operator code base

Reply:
Added `hivemqPlatformServiceAccount.name`